### ISSUE TITLE
Migration Dialog can have broken Continue Button

### ISSUE THREAD

**Issue by mitsuhiko** [getsentry/sentry#6032]
While fighting some fires I rolled back to an earlier Sentry database locally and I was presented with a migration dialog. It asked me for accepting registrations (see #5617 for the change).

Yet the "Continue" button was not functional (grayed out). This seems less than ideal but I'm not entirely sure if that happens under normal operations or if I had some broken state.

**Comment by HazAT** [comment]
So I was playing around with the `needsUpgrade` migration flag and wasn't able to reproduce it. Even after deleting the flag from the features table.
The continue button always worked and it's just disabled if a submit request is currently running
OR
a field fails validation, this specific field you mentioned has a default value of `false` so it's not null and so validates.
tl;dr no able to reproduce and likely to funky state.
